test/features/pagecontent/redirects.js

```diff
@@ -68,6 +68,17 @@ describe('Redirects', function() {
         });
     });
 
+    it('should redirect to commons for missing file pages, dewiki', function() {
+        return preq.get({
+            uri: server.config.hostPort + '/de.wikipedia.org/v1/page/html/Datei:Name.jpg'
+        })
+        .then(function(res) {
+            assert.deepEqual(res.status, 200);
```

Comment:
`preq` is following the redirect, so it's a result for the redirected page. The target of the redirect is checked using the `content-location` header.